src/Components/test/testassets/TestServer/Startup.cs

```diff
@@ -92,6 +84,14 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                     endpoints.MapBlazorHub();
                 });
             });
+
+            app.UseRouting();
+
+            app.UseEndpoints(endpoints =>
+            {
+                endpoints.MapControllers();
+                endpoints.MapRazorPages();
+            });
```

Comment:
I believe I know the answer, but why move this to the bottom?

Reply:
I'm not aware of a real problem here in the way we're using this website, but the old code allowed routing to match (and execute) for URLs that are inside `/prerendered`. That's definitely not intended. I found this while debugging something and decided to clean it up.